### DISCUSSION TITLE
[주차별 리뷰] ※20210125 ~ 20210130 4주차 과제 리뷰※

### DISCUSSION THREAD

**Discussion by dev-LeoKang** [4Moyede/HexatonClass-01#173] [01_주차별 리뷰]
# ※20210125 ~ 20210130 2주차 과제 리뷰※

안녕하세요 강형구, 유혁준 멘토입니다. 금주의 과제 리뷰 올립니다.

## 덕규

- '3. 가독성이 좋은 CODE는 어떤 걸까?' issue 관련 해서 command를 하나 추가하고 싶습니다.  
![image](https://user-images.githubusercontent.com/38308646/106157903-83f19a80-61c6-11eb-86b4-2e7bb2636886.png)  
깔끔한 코드가 됨과 동시에 refactoring이 쉬워진다는 점도 기억하시면 좋을 것 같습니다!  
예를 들면, 3번이 사라지고 6번과 7번 사이에 새로운 내용이 들어간다면? 출력문들을 전부 수정할 필요없이 tuple 만 수정하면 해결이 가능합니다.

## 경훈

- 개체지향프로그래밍 → 객체지향프로그래밍 으로 수정해야할 것 같습니당
- 함수명에 대한 부분을 확실히 안 것 같아 정말 뿌듯하네요 ㅎㅎ

## 재범

- elsf if... false... ture...작성해야 한ㄷ.... 오탈자가 좀 있네요.. 다음부터는 천천히 꼼꼼하게!
- 근데 elif는 이번에 처음 안건가요!!! python에서 else if를 사용하셨다는 말씀이신가요?
- __pycache__ 는 무슨 역할인가요? 왜 생겼을지 생각해서 수업전까지 이슈에 올리기

## 지우

- **3.[Environment] Matplotlib모듈을 설치했음에도 인식하지 못하는 오류,**  이는 path가 잘못되었을 가능성이 큰데, pip install을 한 디렉토리와 가상환경의 디렉토리가 같아야함 → 디렉토리의 정의가 무엇일까요?? 그리고 가상환경의 정의가 무엇일까요?? 수업시간에 물어볼니까 준비해오기~~

## 성준

- 이슈 정리한 마크다운 파일 올리기

**Comment by jiwoo4424** [comment]
확인했습니다

**Comment by ghost** [comment]
제가 정신이 나갔었나 봅니다... 저는 파이썬에서 단 한번도 elif를 쓴 적이 없고 else if도 쓴 적이 없었네요. 왜 else if를 썼다고 생각했는지는 모르겠지만,, 아마 예전에 C 공부할 때 기억과 착각한 모양입니다. 남들은 한번씩 써본 코드를 저는 한 번도 쓰지 않았다는 점에서 자아성찰 하겠습니다.
